scripts/build.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const Bundler = require('parcel-bundler');
```

Comment:
I think we should not reinvent the way we bundle extensions. 

I think that this script should just simply go through the list of extensions that are intended to be hosted on a separate domain. All `build/index.html` files should be committed as they are now. The script would read `version` in package.json of each extension and would push the content of `build/index.html`, tagged by current version, to Netlify or whatever.

Reply:
I strongly disagree with storing build artefacts in git. It's a massive anti-pattern we currently do for Marketplace extensions because we don't have a better alternative. Committed build artefacts have a tendency to get out of sync with their src and you end up with weird commits when the build result changes because of the host environment of the committer.

Reply:
Building extension using `create-contentful-cli` on a fly would also work. I just don't want to have another way to build things.